### PR TITLE
scale down profile picture when window is too small - fixes #85

### LINKED ISSUE #85: Scale profile picture when window is too small
I think we should set the `max-width` to `100%` in order to scale the picture a little bit down if the view becomes to narrow.

screen shot 2016-06-10 at 12 20 17

### PR DESCRIPTION
**Pull request by SamVerschueren**
I'm not sure about the CSS selectors though. Every element has multiple selectors so I'm not sure which one to pick in order to not brake any CSS for other components. Not sure how react calculates the classnames. I just picked the first ones.
## Before

anatine-before
## After

anatine-after

### CODE REVIEW

File: browser.css [hunk 1]
```diff
@@ -56,6 +56,16 @@ img.larrybird {
 	padding-left: 80px !important;
 }
 
+/* scale down the profile picture image in expanded mode and add a margin of 10px */
+div._1qcaWNh- {
+	margin: 10px;
+}
+
```
Comment (paulmolluzzo) [inline browser.css]: Take out this empty line so it's clear that these two blocks are related.

File: browser.css [hunk 2]
```diff
@@ -56,6 +56,16 @@ img.larrybird {
 	padding-left: 80px !important;
 }
 
+/* scale down the profile picture image in expanded mode and add a margin of 10px */
```
Comment (paulmolluzzo) [inline browser.css]: I don't think the comment should reference the exact `margin`. If we ever change the `margin` and forget the comment it'll be confusing.

**Comment by paulmolluzzo** [comment]
> I'm not sure about the CSS selectors though. Every element has multiple selectors so I'm not sure which one to pick in order to not brake any CSS for other components.

I would just ctrl+f for the classes and try to find another place that this combination is being used.  If I search for  `_1qcaWNh- _1sp7Rzwo _3tixQkQf`  I can't find any element with those 3 classes on any page.

With that concept in mind, you could accomplish this same result in this PR with a super specific rule that only styles the `img`:

``` css
div._1qcaWNh-._1sp7Rzwo._3tixQkQf > img._1MjUlO8b._3uSDkk5s {
    box-sizing: border-box;
    padding: 10px;
    max-width: 100%;
    max-height: 100%;
}
```

I'm betting we're unique enough here. If you want to get nuts, `._1u6R0ODP._3tixQkQf` looks like Twitter's generic "modal" overlay wrapper classes, so you could throw that in the selector above. Probably getting overkill though. 😄

**Comment by SamVerschueren** [comment]
Done, works great!

**Comment by sindresorhus** [comment]
Thanks Sam :)

**Comment by SamVerschueren** [comment]
You're welcome! :)

**Pull request merged**